pkgs/tools/filesystems/dduper/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchFromGitHub, python3Packages, btrfs-progs }:
+
+let
+  our-btrfs-progs = btrfs-progs.override {
+    patches = [ patch ];
```

Comment:
Can you overwrite the package instead of adding an optional input?

Reply:
Isn't that dirtier?

Reply:
Having to many options which are not strictly required make the package more complex. I think overwrite is a good option here.